homeassistant/helpers/selector.py

```diff
@@ -73,35 +116,35 @@ class DeviceSelector(Selector):
         }
     )
 
+    def __call__(self, data: Any) -> str:
+        """Validate the passed selection."""
+        return cv.string(data)
+
 
 @SELECTORS.register("area")
 class AreaSelector(Selector):
     """Selector of a single area."""
 
+    name = "area"
+
     CONFIG_SCHEMA = vol.Schema(
         {
-            vol.Optional("entity"): vol.Schema(
-                {
-                    vol.Optional("domain"): str,
-                    vol.Optional("device_class"): str,
-                    vol.Optional("integration"): str,
-                }
-            ),
-            vol.Optional("device"): vol.Schema(
-                {
-                    vol.Optional("integration"): str,
-                    vol.Optional("manufacturer"): str,
-                    vol.Optional("model"): str,
-                }
-            ),
+            vol.Optional("entity"): EntitySelector.CONFIG_SCHEMA,
```

Comment:
This can work today but might not match when we expand the entity selector with a "multi" option and we dont' want to apply that to areas.

Reply:
Then we worry about then